feeds/wifi-ax/mac80211/patches/qca/226-ath11k-set-registered-flag-before-qdss-config.patch

```diff
@@ -0,0 +1,11 @@
+diff -Naur a/drivers/net/wireless/ath/ath11k/core.c b/drivers/net/wireless/ath/ath11k/core.c
+--- a/drivers/net/wireless/ath/ath11k/core.c	2021-09-30 09:44:22.861587609 -0400
++++ b/drivers/net/wireless/ath/ath11k/core.c	2021-09-30 09:45:18.361000898 -0400
+@@ -847,6 +847,7 @@
+ #endif
+ 	ath11k_hif_irq_enable(ab);
+ 
++	set_bit(ATH11K_FLAG_REGISTERED, &ab->dev_flags);
+ 	ath11k_config_qdss(ab);
```

Comment:
Also, please add a detailed comment in the code on why this change is required once my above post is addressed.